mephisto/abstractions/databases/local_database.py

```diff
@@ -255,10 +255,6 @@ def init_tables(self) -> None:
         """
         Run all the table creation SQL queries to ensure the expected tables exist
         """
-        # TODO(#93) maybe raise flag when the schema of existing tables isn't what we expect
-        # it to be?
-        # "How to know that schema changes?"
-        # logger.warning("some message")
```

Comment:
what happens now in the event of a schema change?

Reply:
We're launching 1.0 and I'm not making any schema changes anymore